modules/developers-guide/pages/cli-reference/dfx-parent.adoc

```diff
@@ -62,8 +62,6 @@ For reference information and examples, select an appropriate subcommand.
 [width="100%",cols="<32%,<68%",options="header",]
 |===
 |Command |Description
-|link:dfx-bootstrap{outfilesuffix}[`+bootstrap+`] |Starts the bootstrap server you want to use to serve front-end assets for your project.
-
 |link:dfx-build{outfilesuffix}[`+build+`] |Builds canister output from the source code in your project.
 
 |link:dfx-cache{outfilesuffix}[`+cache+`] |Manages the `+dfx+` cache on the local computer.
```

Comment:
L69:  canisters deployed on a network replica => deployed canisters

Reply:
L84: |link:dfx-ping{outfilesuffix}[`+ping+`] |Sends a response request to a specified {IC} network to determine network connectivity.
If the connection is successful, the {IC} replies with its status. 
=> 
|link:dfx-ping{outfilesuffix}[`+ping+`] |Sends a response request to the Internet Computer blockchain to determine network connectivity.
If the connection is successful, the {IC} replies with its status.

Reply:
L87: Starts a local network replica process => Starts a local canister execution environment

Reply:
L89: the local network replica => the local canister execution environment